cocos/2d/assets/sprite-frame.ts

```diff
@@ -1241,8 +1241,10 @@ export class SpriteFrame extends Asset {
             this._packable = false;
             return;
         }
+        const engineGlobal = typeof globalThis.jsb !== 'undefined' ? (typeof jsb.window !== 'undefined' ? jsb.window : globalThis) : globalThis;
```

Comment:
Please use a more readable format.

Reply:
the expression has been moved into global-exports.ts